lib/plug/debugger.ex

```diff
@@ -549,4 +549,42 @@ defmodule Plug.Debugger do
   defp maybe_merge_dark_styles(style, default_dark_style) do
     Map.put(style, :dark, default_dark_style)
   end
+
+  defp maybe_autolink(message) do
+    splitted = Regex.split(~r/`([^`]+)`/, message, include_captures: true, trim: true)
+
+    Enum.map(splitted, &maybe_format_function_reference/1)
+    |> IO.iodata_to_binary()
+  end
+
+  defp maybe_format_function_reference("`" <> reference = text) do
+    reference = String.trim_trailing(reference, "`")
+
+    with true <- function_reference?(reference),
+         {:ok, m, f, a} <- get_mfa(reference),
+         url when is_binary(url) <- get_doc(m, f, a, Application.get_application(m)) do
+      ~s[<a href="#{url}" target="_blank">`#{h(reference)}`</a>]
+    else
+      _ -> h(text)
+    end
+  end
+
+  defp maybe_format_function_reference(text), do: h(text)
+
+  defp function_reference?(text) do
+    Regex.match?(~r/^[A-Z][A-Za-z0-9_.]+\.[a-z][A-Za-z0-9_!?]*\/\d+$/, text)
```

Comment:
We should probably merge this regex into the previous one, so we only parse `...` that already match this form?